docs/how-to-setup-freecodecamp-locally.md

```diff
@@ -213,6 +213,9 @@ Before you can run the application locally, you will need to start the MongoDB s
 > [!NOTE]
 > Unless you have MongoDB running in a setup different than the default, the URL stored as the `MONGOHQ_URL` value in the `.env` file should work fine. If you are using a custom configuration, modify this value as needed.
 
+> [!NOTE]
+> If you followed along with the [Windows 10 via WSL2 Setup Guide](how-to-setup-wsl.md), then you should be able to skip this step if the MongoDB server from that guide is already running. You can confirm this by checking that you can reach `http://localhost:27017` on your local machine.
+
```

Comment:
```suggestion
> [!NOTE]
> Unless you have MongoDB running in a setup different than the default, the URL stored as the `MONGOHQ_URL` value in the `.env` file should work fine. If you are using a custom configuration, modify this value as needed.
>
> If you followed along with the [Windows 10 via WSL2 Setup Guide](how-to-setup-wsl.md), then you should be able to skip this step if the MongoDB server from that guide is already running. You can confirm this by checking that you can reach `http://localhost:27017` on your local machine.
```

Two note-boxes together looks off - can we combine them?

Reply:
I made the requested change, here's a quick screenshot of how it looks
![screenshot-freecodecamp_updated](https://user-images.githubusercontent.com/6621379/193117131-f24b5873-74b2-4b35-b071-b4ea33e9e820.png)